nginx.conf

```diff
@@ -46,7 +48,13 @@ http {
             add_header Content-Security-Policy $CSP;
             root /app;
             index index.html;
-            try_files $uri /index.html;
+            try_files $uri $uri/ /index.html;
+        }
+
+        location /storybook {
```

Comment:
Uhm, thanks, but I don't want to put storybook to production server))

Reply:
That's the wrong push from a different task, it will be removed here, sorry for the mistake